client/src/redux/failed-updates-epic.js

```diff
@@ -19,13 +19,25 @@ import {
 } from './';
 import postUpdate$ from '../templates/Challenges/utils/postUpdate$';
 import { isGoodXHRStatus } from '../templates/Challenges/utils';
+import { backEndProject } from '../../utils/challengeTypes';
 
 const key = 'fcc-failed-updates';
 
 function delay(time = 0, fn) {
   return setTimeout(fn, time);
 }
 
+function isSubmitable(failure) {
+  // check if backend challenges have a solution
+  if (
```

Comment:
More concise way of writing this.
```suggestion
    return failure.payload.challengeType !== backEndProject || failure.payload.solution;
```

Reply:
I was thinking of adding more test for other project types, but I like your version much better.